pages/k8s/ipv6.md

```diff
@@ -0,0 +1,156 @@
+---
+wrapper_template: "kubernetes/docs/base_docs.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "IPv6"
+  description: Using IPv6 with Charmed Kubernetes.
+keywords: juju, network, networking
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: ipv6.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+As of Kubernetes 1.19, support for IPv6 is in beta and dual-stack is in alpha.
+Charmed Kubernetes can enable these by including IPv6 CIDRs in the `cidr`
+config for the Calico charm and the `service-cidr` config for the Kubernetes
+Master charm.  These can contain up to two comma-separated values, with the
+first CIDR in the list being the preferred family for pods, or the default for
+services.  You can override the `IPFamily` for a service when creating it.
+
+<div class="p-notification--positive"><p markdown="1" class="p-notification__response">
+<span class="p-notification__status">Note:</span>
+Calico is the only CNI which supports IPv6 at this time.
+</p></div>
+
+## Example
+
+You can use the following overlay file ([download it here](asset-ipv4-ipv6-overlay])
+to enable IPv4 preferred dual-stack to the [kubernetes-calico][] bundle:
```

Comment:
we don't 'use' those cni specific bundles in the docs, so really it should be in conjunction with the calico overlay.

Reply:
The docs page for Calico [says to use that bundle](https://ubuntu.com/kubernetes/docs/cni-calico#deploying-charmed-kubernetes-with-calico).  Maybe we need to update that page to use the overlay as well?

Reply:
yes. I thought there was an existing issue for that but I can't find it. I will make a new one